tests/ToolInfoTest.php

```diff
@@ -23,11 +23,61 @@
 {
     public function testGetVersion()
     {
-        $this->assertInternalType('string', ToolInfo::getVersion());
+        $toolInfo = new ToolInfo();
+        $this->assertInternalType('string', $toolInfo->getVersion());
     }
 
     public function testIsInstallAsPhar()
     {
-        $this->assertFalse(ToolInfo::isInstalledAsPhar());
+        $toolInfo = new ToolInfo();
+        $this->assertFalse($toolInfo->isInstalledAsPhar());
+    }
+
+    /**
+     * @dataProvider provideScriptName
+     *
+     * @param mixed $scriptName
+     */
+    public function testIsInstalledByComposer($scriptName)
+    {
+        $phpunitBin = $this->getRootDir().'/vendor/bin/phpunit';
+        if (!is_file($phpunitBin)) {
+            $this->markTestSkipped(sprintf('Binary "%s" is necessary to emulate PHP-CS-Fixer installed through composer', $phpunitBin));
+        }
+
+        if (!is_link($scriptName)) {
+            $this->markTestSkipped(sprintf('OS "%s" doesn\'t support the way the "%s" symlink is written', PHP_OS, $scriptName));
```

Comment:
can we create and remove the symlink(s) in the setup/teardown of the tests or would that complicate the tests to much?